include/zmq.h

```diff
@@ -656,6 +656,8 @@ ZMQ_EXPORT void zmq_threadclose (void *thread_);
 #define ZMQ_MULTICAST_LOOP 96
 #define ZMQ_ROUTER_NOTIFY 97
 #define ZMQ_XPUB_MANUAL_LAST_VALUE 98
+#define ZMQ_SOCKS_USERNAME 99
```

Comment:
these 2 need to be duplicated in src/zmq_draft.hpp otherwise the build will break

Reply:
Ah ok. Sorry.
By the way, is it the right way to proceed, i.e. adding new sock opts as draft first?

Reply:
yes

Reply:
Done.